_config.yml

```diff
@@ -1,15 +1,17 @@
 name: ROS2 Design
 description: "Distilled design documents related to the ROS 2.0 effort"
 
-baseurl: http://design.ros2.org
+baseurl: ""
```

Comment:
It's not necessary, but I think it's better to have as empty since we reference it throughout the code. It's a value which can be tuned so it's better to have the introspection that it's empty.